src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Advanced CoreDNS Configuration"
+description = "Here we describe how you can customize the configuration of the managed CoreDNS service in your clusters"
+date = "2018-10-30"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+# Advanced CoreDNS Configuration
+
+The [CoreDNS addon](https://github.com/coredns/coredns) running inside your cluster has additional configuration options and features that can be customized.
+
+You can customize two of these configuration options on a per cluster basis through a ConfigMap inside your clusters. The ConfigMap is named `coredns-user-values` and is located in the `kube-system` namespace.
```

Comment:
@TheoBrigitte is the name and namespace right?

Reply:
correct :heavy_check_mark: